packages/wrangler/src/entry.ts

```diff
@@ -48,6 +48,22 @@ export async function getEntry(
     directory,
     args.format ?? config.build?.upload?.format
   );
+
+  if (format === "service-worker" && hasDurableObjectImplementations(config)) {
+    const errorMessage =
+      "You seem to be trying to use Durable Objects with Service Worker syntax.";
+    const addScriptName =
+      "You can use Durable Objects defined in other (Module) Workers from a Service Worker by specifying a `script_name` in your wrangler.toml,where `script_name` is the name of the worker that implements that Durable Object. For example:";
+    const addScriptNameExamples = generateAddScriptNameExamples(config);
+    const migrateText =
+      "Alternatively, migrate your worker to Module format to implement a new Durable Object:";
```

Comment:
What is the convention? Should this be `ES Module format`?

Reply:
[It's unclear](https://developers.cloudflare.com/workers/learning/migrating-to-module-workers/) to me :(

Reply:
Either works for me.